publish-release.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+set -ex
```

Comment:
be even stricter
https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425#set--e--u--x--o-pipefail